github-for-jira.sd.yml

```diff
@@ -6,11 +6,25 @@ notifications:
   email: fusion-arc@atlassian.com
 
 compose:
+  pgbouncer:
+    image: docker.atl-paas.net/sox/micros/pgbouncer
+    tag: 1.15.0
+    command: [ "database" ]
+    environment:
+      PGBOUNCER_POOL_MODE: transaction
+      PGBOUNCER_DEFAULT_POOL_SIZE: 15 # Max scale up:
+                                      #   (10 web servers + 6 workers) * 15 pool size  = 240 connections
+                                      # Normal business:
+                                      #   (5 web servers + 3 workers) * 15 pool size = 120 connections
+      PGBOUNCER_SERVER_IDLE_TIMEOUT: 60
+      PGBOUNCER_MAX_CLIENT_CONN: 1000
```

Comment:
List of available options: https://hello.atlassian.net/wiki/spaces/MICROS/pages/167213858/HOWTO+Run+pgbouncer+as+a+sidecar

Reply:
https://hello.atlassian.net/wiki/spaces/MICROS/pages/167213858/HOWTO+Run+pgbouncer+as+a+sidecar

Reply:
https://sourcegraph-frontend.internal.shared-prod.us-west-2.kitt-inf.net/bitbucket.org/atlassianlabs/prlinks/-/blob/evzijst-prlinks.sd.yml sample prod config